### ISSUE TITLE
git-pr-release installed with rbenv does nothing

### ISSUE THREAD

**Issue by hkurokawa** [motemen/git-pr-release#29]
# Environment
- git-pr-release 0.8.0
- rbenv 1.1.2
- ruby 2.3.7
- macOS 10.13.6

# Description
When I install git-pr-release >= 0.7.0 with rbenv, It seems running `git-pr-release` does nothing.

```bash
$ which ruby
/Users/hiroshi/.rbenv/shims/ruby
$ which gem
/Users/hiroshi/.rbenv/shims/gem
$ gem install git-pr-release
Fetching: git-pr-release-0.8.0.gem (100%)
Successfully installed git-pr-release-0.8.0
Parsing documentation for git-pr-release-0.8.0
Installing ri documentation for git-pr-release-0.8.0
Done installing documentation for git-pr-release after 0 seconds
1 gem installed
$ DEBUG=true git-pr-release
$
```

# Steps to reproduce
1. Install rbenv
2. Install ruby with `rbenv install 2.3.7`
3. Install git-pr-release with `gem install git-pr-release`
4. Run `git-pr-release`

# Result
At step 4, nothing is displayed and the command finishes silently

# Expected result
At step 4, something like below output is shown:

```bash
$ DEBUG=true git-pr-release
Executing `git config remote.origin.url`
Executing `git config -f .git-pr-release pr-release.branch.production`
Executing `git config -f .git-pr-release pr-release.branch.staging`
Repository:        hkurokawa/dummy-repo
Production branch: release
Staging branch:    master
Executing `git config -f .git-pr-release pr-release.token`
Executing `git config pr-release.token`
Executing `git remote update origin`
Executing `git log --merges --pretty=format:%P origin/production..origin/master`
Executing `git ls-remote origin refs/pull/*/head`
No pull requests to be released
```

# Remarks
I believe this issue was introduced by https://github.com/motemen/git-pr-release/pull/25/commits/d1e5052cd4b9dd9cd7841cb5f38e06b0de50fe91 and this issue is related to #27 

I don't know whether adding `if` statement to make sure the script is not executed when it is just loaded is the idiomatic way.  But it seems `~/.rbenv/versions/2.3.7/bin/git-pr-release` does load the script as below.  So I am wondering if there might be no way to tell apart script loading from script execution (as rbenv *loads* the script to run).

```ruby
load Gem.bin_path('git-pr-release', 'git-pr-release', version)
```

If you need further information or have a question, feel free to ask me.

Thanks.

**Comment by onk** [comment]
Sorry for no response.

I prefer to extract to cli module like #30.

**Comment by hkurokawa** [comment]
No problem.  Thank you for help and feel free to close the issue when a new release is shipped.

**Comment by hkurokawa** [comment]
Hi,

Still suffering from this problem.  Is there any idea on the timeline of the release?

**Comment by onk** [comment]
Released as v1.0.1.
